src/System.Management.Automation/engine/CommandSearcher.cs

```diff
@@ -528,6 +505,65 @@ private CommandInfo GetNextFromPath()
             return result;
         }
 
+        /// <summary>
+        /// Gets the next path using WildCards.
+        /// </summary>
+        /// <param name="command">
+        /// The command to search for.
+        /// </param>
+        /// <param name="provider">The provider that the command was found in.</param>
+        /// <returns>
+        /// A collection of full paths to the commands which were found.
+        /// </returns>
+        private Collection<string> GetNextFromPathUsingWildcards(string command, out ProviderInfo provider)
+        {
+            try
+            {
+                Provider.CmdletProvider providerInstance;
+                return _context.LocationGlobber.GetGlobbedProviderPathsFromMonadPath(command, false, out provider, out providerInstance);
```

Comment:
It seems we don't use `providerInstance`.
```suggestion
                return _context.LocationGlobber.GetGlobbedProviderPathsFromMonadPath(command, false, out provider, out _);
```